src/Utilities.php

```diff
@@ -33,7 +35,23 @@ public static function getEntityDataCount($entity_type, $bundle = NULL) {
     }
 
     $query->addMetaData('account', user_load(1));
-    return (int) $query->count()->execute();
+    try {
+      return (int) $query->count()->execute();
+    }
+    catch (\Exception $e) {
+      // Use dt() with "!" instead of "@" placeholders, to avoid html in cli
+      // output.
+      // @todo Is this the recommended way to do this?
+      drush_log(
+        dt(
+          "Exception while counting !type entities:\n!message",
+          [
+            '!type' => ($bundle === NULL) ? $entity_type : "$entity_type:$bundle",
+            '!message' => $e->getMessage(),
+          ]),
+        LogLevel::WARNING);
+      return -1;
```

Comment:
Thanks for this @donquixote 
It looks good to me, though I'd personally format it as follows:
```
drush_log(dt('Exception while counting @type entities: !message', [
  '@type' => ($bundle === NULL) ? $entity_type : "$entity_type:$bundle",
  '!message' => $e->getMessage(),
]), LogLevel::WARNING);
```

There is also `drush_set_error()` but maybe we should only throw a warning, not error?
I don't have a strong opinion on this.

Reply:
I don't have a strong opinion about the formatting.

For the warning vs error, I would say: Warning if operation can proceed afterwards, error if we have to give up and stop.